### ISSUE TITLE
coredump: Dont delete coredumps by age

### ISSUE THREAD

**Issue by nolange** [systemd/systemd#15712]
I just realized that `/lib/tmpfiles.d/systemd.conf` contains a line

```
d /var/lib/systemd/coredump 0755 root root 3d
```

I really dont want coredumps to go ever, and the issue is that there aint a clean way to do that.

1.  I could edit `/lib/tmpfiles.d/systemd.conf` - everytime I update systemd
2.  I could create a file `/etc/tmpfiles.d/systemd.conf` replacing the line, but this is fragile since it should contain everything else from the file in `/lib`
3.  create a new file, lexicographically before systemd.conf. This will then then give warnings every time systemd-tmpfiles is invoked.

In short, I lack a clean method to just affect the coredump settings. IMHO conf files containing "cleaning" information should either be separate (like `clean-systemd-coredump.conf`), or there should be a way to mark lines as an "override", like adding a + in front.

```
# Apply the overrides (in the usual order) after the normal lines,
# not warning that there is a duplicate (maybe warn if there is not?).
+d /var/lib/systemd/coredump 0755 root root -
```

**Comment by poettering** [comment]
Duplicate of #8809
